src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
```

Comment:
You shouldn't set margins to blocks, do it in a parent block instead

Reply:
i made it in calendar block for center it